packages/mobile/src/components/details-tile/DetailsTileHasAccess.tsx

```diff
@@ -1,24 +1,31 @@
-import type { ReactNode } from 'react'
 import { useCallback } from 'react'
 
 import type { PremiumConditions, User } from '@audius/common'
-import { usePremiumConditionsEntity } from '@audius/common'
+import {
+  formatUSDC,
+  isPremiumContentCollectibleGated,
+  isPremiumContentFollowGated,
+  isPremiumContentTipGated,
+  isPremiumContentUSDCPurchaseGated,
+  usePremiumConditionsEntity
+} from '@audius/common'
 import type { ViewStyle } from 'react-native'
-import { View, Text } from 'react-native'
+import { View } from 'react-native'
 
+import IconCart from 'app/assets/images/iconCart.svg'
 import IconCollectible from 'app/assets/images/iconCollectible.svg'
 import IconSpecialAccess from 'app/assets/images/iconSpecialAccess.svg'
-import IconUnlocked from 'app/assets/images/iconUnlocked.svg'
-import { useLink } from 'app/components/core'
+import { useLink, Text, LockedStatusBadge } from 'app/components/core'
 import UserBadges from 'app/components/user-badges'
 import { useNavigation } from 'app/hooks/useNavigation'
 import { flexRowCentered, makeStyles } from 'app/styles'
-import { useColor, useThemePalette } from 'app/utils/theme'
+import { useColor } from 'app/utils/theme'
 
 const messages = {
   unlocked: 'UNLOCKED',
   collectibleGated: 'COLLECTIBLE GATED',
   specialAccess: 'SPECIAL ACCESS',
+  USDCPurchaseTitle: 'Pay to Unlock',
```

Comment:
nit: can probably remove usdc reference from this variable to make things easier? (`payToUnlock: 'Pay to Unlock'`)